src/backend/aspen/workflows/nextstrain_run/build_plugins/type_plugins.py

```diff
@@ -279,20 +279,36 @@ def apply_filters(config, subsampling, template_args):
         include_arguments_in_filters = True
         lineage_field = "pango_lineage"
 
-    filter_map = {"filter_start_date": "min_date", "filter_end_date": "max_date"}
-    for filter_name, yaml_key in filter_map.items():
-        value = template_args.get(filter_name)
-        if not value:
-            continue  # This filter isn't set, skip it.
+    min_date = template_args.get("filter_start_date")
+    if min_date:
         # Support date expressions like "5 days ago" in our cron schedule.
-        value = dateparser.parse(value).strftime("%Y-%m-%d")
+        min_date = dateparser.parse(min_date).strftime("%Y-%m-%d")
         if include_arguments_in_filters:
             subsampling["group"][
-                yaml_key
-            ] = f"--{yaml_key.replace('_', '-')} {value}"  # ex: --max-date 2020-01-01
+                "min_date"
+            ] = f"--{"min_date".replace('_', '-')} {min_date}"  # ex: --max-date 2020-01-01
         else:
-            subsampling["group"][yaml_key.replace("_", "-")] = str(
-                value
+            subsampling["group"]["min_date".replace("_", "-")] = str(
+                min_date
+            )  # ex: max-date: 2020-01-01
+
+    max_date = template_args.get("filter_end_date")
+    if max_date:
+        # Support date expressions like "5 days ago" in our cron schedule.
+        max_date = dateparser.parse(max_date).strftime("%Y-%m-%d")
+        if include_arguments_in_filters:
+            subsampling["group"][
+                "max_date"
+            ] = f"--{"max_date".replace('_', '-')} {max_date}"  # ex: --max-date 2020-01-01
+            subsampling["international_serial_sampling"][
+                "max_date"
+            ] = f"--{"max_date".replace('_', '-')} {max_date}"  # ex: --max-date 2020-01-01
+        else:
+            subsampling["group"]["max_date".replace("_", "-")] = str(
+                max_date
+            )  # ex: max-date: 2020-01-01
+            subsampling["group"]["max_date".replace("_", "-")] = str(
+                max_date
             )  # ex: max-date: 2020-01-01
 
     pango_lineages = template_args.get("filter_pango_lineages")
```

Comment:
(this is for doc writing) how is mpox lineage filter added?

Reply:
@ehoops-cz I think we can support this on the backend, but I'm not sure if the designs included this?

Reply:
I found Janeece's design doc. It says "keep lineage for HepC" which prob means not for mpox. 
<img width="763" alt="image" src="https://user-images.githubusercontent.com/20667188/216707627-db4e1dbd-7ae8-45a2-a8c7-4668f776aa31.png">